tests/Get-DbaDbSharePoint.Tests.ps1

```diff
@@ -15,29 +15,47 @@ Describe "$CommandName Unit Tests" -Tag 'UnitTests' {
 
 Describe "$commandname Integration Tests" -Tag "IntegrationTests" {
     BeforeAll {
+        $skip = $false
```

Comment:
I never *really* got to use this function and I fear it spun up by potatoqualitee's period on working tirelessly on sharepoint farms. That being said I'm seeing multiple reports about either SQLPS loading DLLs that interfere with us and also DacFx-related issues (many cycling around core-noncore dilemma). That being said I trust you on this change, maybe @potatoqualitee  can chime in.

Reply:
i dont remember anything and also trust the change 😅